bylaws/2-cf-membership.md

```diff
@@ -20,22 +16,28 @@ The following section outlines the different types of membership and the process
 
 A **project** within the CF represents collaborative innovation in open source technology. These projects are dynamic collectives of individual contributors, functioning as self-organizing units rather than formal legal entities.[^1]
 
-**Flexible and Inclusive Structure:** Projects can vary in size and scope, ranging from individual repositories to entire collections of repositories. Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+**Flexible and Inclusive Structure:** Projects can vary in size and scope, ranging from individual repositories to entire collections of repositories.
+Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+While the CF uses GitHub as the primary platform for collaboration, projects may be hosted on other platforms as well.
```

Comment:
Is there some sort of requirement that such a platform must meet? Like, from a project continuity point of view, it should not be enough to have the GH repo hosted on a server in my bedroom, nor even on a server hosted by a small company.

Reply:
I want to include those specifics in project on-boarding + fiscal hosting agreement, rather than in the bylaws.

Reply:
OK